packages/api-reference/src/features/Search/SearchModal.vue

```diff
@@ -95,6 +86,26 @@ function onSearchResultClick(entry: FuseResult<FuseData>) {
   }
 }
 
+// Scroll to the currently selected result
+watch(selectedSearchResult, (index) => {
+  const newResult = searchResultsWithPlaceholderResults.value[index]
+
+  document.getElementById(newResult.item.href)?.scrollIntoView({
+    behavior: 'smooth',
+    block: 'start',
```

Comment:
Do we actually want this to be nearest?

Reply:
this is currently up for debate haha anthony is on team center, idc really

Reply:
Well if we do any of the positioned ones then it moves every time you press an arrow key even if it doesn't need to.

**`block: 'start'`**:
 
https://github.com/user-attachments/assets/c659dfd9-a0ea-4688-9166-276e10f93401

**`block: 'nearest'`**:

https://github.com/user-attachments/assets/b2b83e79-cb82-4ca7-9ffa-acf1d10fce97

Reply:
I'm in for nearest across the board. Will wait for the others to confirm then merge. Good call!